### DISCUSSION TITLE
Can't get rid of modification names in the UniMod format

### DISCUSSION THREAD

**Discussion by diabeginner01** [vdemichev/DiaNN#535] [Q&A] [answered]
Hi Vadim,

I'm trying force DIA-NN to display the full names of the modifications (e.g. Carbamidomethylation) instead of the UniMod code. I've tried to use both of the following commands:
--original-mods
--full-unimod
And DIA-NN also states during the processing that:
"DIA-NN will not attempt to convert library modifications to the UniMod format"

Yet, I end up with the results still with UniMod codes...I'm doing library-free search, so I am wondering does this command apply only if I have a preannotated spectral library. How do I get DIA-NN to display the full names of the modifications when doing library-free search?

**Comment by vdemichev** [comment, marked as answer]
--original-mods just preserves the mods with which modifications are declared. So if they are declared with a unimode code, then it will be the same format in the output. Btw, in general, using find-replace on DIA-NN's report is very straightforward in both R or Python.
